fixed empty output from runtime executor in CPU flow

zonglinpeng opened the pull request:
Summary:
- add missing dependency dtype_util.cpp
- it was missing etrecord. And dump-output is not toggled in executor.
- Also enable runtime logging

Differential Revision: D64272605

mcremon-meta commented on backends/cadence/reference/operators/CMakeLists.txt:
```diff
@@ -52,6 +52,7 @@ set(_aten_ops__srcs
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_expand_copy.cpp"
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_gelu.cpp"
     "${EXECUTORCH_ROOT}/kernels/portable/cpu/op_empty.cpp"
+    "${EXECUTORCH_ROOT}/kernels/portable/cpu/util/dtype_util.cpp"
```
maybe move it up with the other utils?

mcremon-meta commented on backends/cadence/aot/export_example.py:
```diff
@@ -14,7 +14,7 @@
 from executorch.backends.cadence.aot.compiler import (
     convert_pt2,
-    export_to_cadence,
+    export_to_cadence_edge_executorch,
```
is that API new? I don't see it in the changes

zonglinpeng replied:
good catch. now ported.

mcremon-meta commented on backends/cadence/aot/compiler.py:
```diff
@@ -197,11 +204,12 @@ def export_to_edge(
 # apply passes specific to Cadence DSP execution. Return both to print the
 # differences.
-def export_to_cadence(
+def export_to_cadence_edge_executorch(
```
do we need "edge"?

zonglinpeng committed:
fixed empty output from runtime executor in CPU flow (#6184)

Summary:

- add missing dependency dtype_util.cpp
- it was missing etrecord. And dump-output is not toggled in executor.
- Also enable runtime logging

Reviewed By: Vysarat, mcremon-meta

Differential Revision: D64272605
